test/createMockFile.ts

```diff
@@ -1,4 +1,4 @@
-export const createMockFile = (name: string = 'mock.txt', size: number = 1024, mimeType: string = 'plain/txt') => {
```

Comment:
Not sure it's necessary to change the defaults here - just pass in the docx where needed